crates/forge/bin/main.rs

```diff
@@ -1,6 +1,8 @@
 //! The `forge` CLI: build, test, fuzz, debug and deploy Solidity contracts, like Hardhat, Brownie,
 //! Ape.
 
+#![cfg_attr(not(test), warn(unused_crate_dependencies))]
```

Comment:
sorry i should've specified to do it only in lib.rs, as main.rs will have unnecessary warnings

Reply:
No worries, re-added to main files. Still seeing the false positives where removing the warned dependencies throws errors as the dependencies are used in test files and the linter specifies not(test). In this case should I add `use toml as _;` etc. in the crate root to silence the warnings as `make pr` is currenly failing due to the linting?

Reply:
in that case you need to move them to dev-dependencies

Reply:
addressed, thanks!